neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -65,20 +70,16 @@ public UInt256 Hash
             sizeof(long) +              //NetworkFee
             sizeof(uint) +              //ValidUntilBlock
             Attributes.GetVarSize() +   //Attributes
-            Witness.Size;               //Witnesses
+            Witnesses.GetVarSize();     //Witnesses
 
         public void CalculateFees()
         {
             if (Sender is null) Sender = UInt160.Zero;
             if (Attributes is null) Attributes = new TransactionAttribute[0];
-            if (Witness is null) Witness = new Witness
-            {
-                InvocationScript = new byte[65],
-                VerificationScript = new byte[39]
-            };
+            if (Witnesses is null) Witnesses = new Witness[0];
```

Comment:
When we add `Witnesses`, the fee will be increased. It is a problem to calculate the fee precisely.

Reply:
Limiting some opcodes could perhaps help resolving this calculation, as most of them will not be necessary on verification scripts. We can try to solve this now, or leave it for a later update too.